doc/themes/the-things-stack/layouts/index.html

```diff
@@ -1,60 +1,75 @@
-
-{{ $enddevices := dict "title" "End Devices" "button" "Learn More" "url" (.Site.GetPage "devices").RelPermalink "text" "Learn how to configure and manage LoRaWAN End Devices in The Things Stack."}}
-
-{{ $gateways := dict "title" "Gateways" "button" "Learn More" "url" (.Site.GetPage "gateways").RelPermalink "text" "Learn to connect and manage various LoRaWAN Gateways on The Things Stack."}}
-
-{{ $data := dict "title" "Integrations" "button" "Learn More" "url" (.Site.GetPage "integrations").RelPermalink "text" "Learn how to work with End Device data on The Things Stack and how to integrate with external systems."}}
+<div>
 
-{{ $tts := dict "title" "The Things Stack" "button" "Learn More" "url" (.Site.GetPage "the-things-stack").RelPermalink "text" " Learn about the different flavours of The Things Stack and how to install/access them."}}
+  <section class="section-doc-home-page-hero ttui-container grid pt-ls-xl pb-ls-xl">
 
-{{ $reference := dict "title" "Reference" "button" "Learn More" "url" (.Site.GetPage "reference").RelPermalink "text" "Curated list of advanced topics for The Things Stack. Leverage the full customization capabilities of The Things Stack."}}
-
-{{ $cards := slice $quickstart $enddevices $gateways $data $tts $reference }}
-
-<div class="section">
-  <div class="container is-max-desktop">
-    <div class="columns" style="flex-wrap: wrap" >
-      {{ range $cards}}
-        <div class="column is-flex">
-          <div class="card is-fullwidth">
-            <div class="card-content">
-              <div class="title-content">
-                <h2 class="title is-size-5 has-text-centered">{{ .title }}</h2>
-                <div class="content">
-                    {{ .text | markdownify }}
-                </div>
-              </div>
-              <div class="buttons is-centered">
-                <a href="{{ .url }}" class="button is-primary is-outlined">{{ .button }}</a>
-              </div>
+    <div class="item-5 md:item-12 grid sm:mb-ls-l">
+      <div class="mb-ls-l item-12 md:item-6 sm:item-8 xs:item-12">
+        <p class="bg-tts-primary-main d-inline-block c-white ttui-body p-sides-cs-s pt-cs-xxs pb-cs-xxs br-xxl mb-cs-m">v{{- .Site.Params.version -}}</p>
+        <h1 class="mb-cs-s c-interactive-03">The Things Stack Documentation</h1>
+        <p class="ttui-body fs-l mb-cs-xl">The Things Stack is a robust, yet flexible LoRaWAN® Network Server that caters to the needs of demanding LoRaWAN deployments, from covering the essentials to advanced security configurations and device life cycle management.</p>
+        <div class="d-flex gap-cs-m">
+          <a href="https://www.thethingsindustries.com/stack/plans">
+            <button class="ttui-btn ttui-btn--primary">Choose your plan</button>
+          </a>
+          <a href="https://www.thethingsindustries.com/contact">
+            <button class="ttui-btn ttui-btn--secondary">Talk to an expert -></button>
+          </a>
+        </div>
+      </div>
```

Comment:
Since we have the `Get The Things Stack` button in the header, I'm not sure of this Call to Action that leads to `thethingsindustries.com` in the main view area. I think it distracts from the purpose of the documentation. 
<img width="1887" alt="Screenshot 2023-08-01 at 11 49 30" src="https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/13186113/2daa051d-3013-4cf9-a649-7904bab57e82">

Can we drop these two buttons from the main area?

See the [stripe documentation](https://stripe.com/docs) for inspiration.

Reply:
@KrishnaIyer https://github.com/TheThingsIndustries/thethingsindustries.com/issues/662 linking to this which is some similar thoughts for all the headers and jumbotrons. This is all need to be discussed with @wienke and @rish

Reply:
Based on an offline discussion, this has been resolved.